functions-common

```diff
@@ -1351,6 +1351,8 @@ function uninstall_package {
         sudo ${YUM:-yum} remove -y "$@" ||:
     elif is_suse; then
         sudo zypper remove -y "$@" ||:
+    elif is_clearlinux; then
+	echo "don't uninstall packages on clearlinux now"
```

Comment:
In clearlinux, we have to skip uninstall, because if we pass a bundle name here, the whole bundle will be uninstalled, if we pass a pkg name here, bundle-delete cannot find it which will generate an error.

Reply:
I can live with this as the only OS packages we normally try to uninstall are the ones that have historically been a problem to restart, specifically erlang and rabbitmq.